<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb10">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb50">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb100">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pb500">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>500</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCoffee">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Coffee(100)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbTea">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Tea(150)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCoke">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>360</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Coke(200)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbReset">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>490</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
